### PR TITLE
convert the troubleshotting.md to proper md

### PR DESCRIPTION
**Pull request by weshayutin**
* mostly just converting to markdown
* moved some links around, imho better organization.
* not perfect but better

## Why the changes were made

* the old page was too ugly

### CODE REVIEW

**Review by kaovilai — COMMENTED** [review]
FYI it was already markdown. Just that markdown also accept HTML headers which Emily used to center headings.

Without HTML headers, there is no way to align center the headers. Cosmetic reasons.

File: docs/TROUBLESHOOTING.md [hunk 1]
```diff
@@ -1,80 +1,76 @@
...
-7. [OpenShift ROSA STS and OADP installation](https://github.com/rh-mobb/documentation/blob/main/content/docs/misc/oadp/rosa-sts/_index.md)
-
-
-<hr style="height:1px;border:none;color:#333;">
-
-<h1 align="center">Debugging Failed Backups<a id="backup"></a></h1>
-
-This section includes steps to debug a failed backup. For more specific issues related to Restic/CSI/volume snapshots check out the following section. Both the OpenShift commands and native Velero commands are useful in troubleshooting.
+1. [OADP FAQ](https://access.redhat.com/articles/5456281)
+1. [OADP Official Troubleshooting Documentation](https://docs.openshift.com/container-platform/latest/backup_and_restore/application_backup_and_restore/troubleshooting.html)
```
Comment (kaovilai) [inline docs/TROUBLESHOOTING.md:8]: Should say when to use FAQ vs reading official troubleshooting doc

Comment (weshayutin) [reply docs/TROUBLESHOOTING.md:8]: I don't really have a universal rule as long as what we had off has links to official doc and faq.

**Comment by weshayutin** [comment]
> FYI it was already markdown. Just that markdown also accept HTML headers which Emily used to center headings.
> 
> Without HTML headers, there is no way to align center the headers. Cosmetic reasons.

I know, it's just ugly. We have real doc now. This can just be standard markdown

**Review by kaovilai — APPROVED** [review]

**Review by shubham-pampattiwar — APPROVED** [review]

**Comment by weshayutin** [comment]
hrm.. @kaovilai what's up w/ the "waiting for status to be reported"

**Comment by kaovilai** [comment]
🤷 maybe a job forgets to run.. if this is md only we can override anyway

**Comment by kaovilai** [comment]
/override ci/prow/4.17-ci-index ci/prow/4.17-images ci/prow/4.18-dev-preview-ci-index ci/prow/4.18-dev-preview-images

**Comment by kaovilai** [comment]
and tide already says in merge pool.. whatever that means

**Comment by kaovilai** [comment]
oh I think it's because there was some other PR tide wants to merge first but was failing

**Comment by kaovilai** [comment]
/override ci/prow/4.14-ci-index ci/prow/4.14-images ci/prow/4.15-ci-index ci/prow/4.15-images ci/prow/4.16-ci-index ci/prow/4.16-images ci/prow/images ci/prow/unit-test

**Pull request merged**